### ISSUE TITLE
Error in converting Pytorch Faster-RCNN model to TensorRT

### ISSUE THREAD

**Issue by Alex-Derhacobian** [NVIDIA/TensorRT#1429]
## Description

I am trying to build a TensorRT engine from a PyTorch Faster-RCNN model pretrained on Imagenet with a ResNet-50-FPN backbone. (see [https://pytorch.org/vision/stable/models.html#torchvision.models.detection.fasterrcnn_resnet50_fpn](url)). I am able to save the model as an onnx file, but running trtexec fails. See below for the exact error received:

## Environment

**TensorRT Version**: 8.0.1
**NVIDIA GPU**: TITAN V
**NVIDIA Driver Version**: 460.39
**CUDA Version**: 11.0
**CUDNN Version**: 8.0.3
**Operating System**: Ubuntu 16.04.2
**Python Version (if applicable)**: 3.8.10
**Tensorflow Version (if applicable)**: N/A
**PyTorch Version (if applicable)**: 1.9.0
**Torchvision Version (if applicable)**: 0.10.0

## Relevant Files

For standard error output and full steps to reproduce the error, see [https://github.com/Alex-Derhacobian/tensorrt_frcnn_bug ](url)

## Steps To Reproduce

(for more in depth reproduction steps, see the repo linked above). 
I save the Pytorch model to an onnx file as follows:
```
model = torchvision.models.detection.fasterrcnn_resnet50_fpn(pretrained = True) 
onnx_opset_version = 11 
onnx_out_fname = f'frcnn_opset={onnx_opset_version}.onnx'
torch.onnx.export(model, 
            inp, 
            onnx_out_fname, 
            output_names = ['boxes', 'labels', 'scores'], 
            verbose = True, 
            export_params = True,
            opset_version = onnx_opset_version)
```
To check that the model was successfully saved as an onnx file, I run
```
model = onnx.load(onnx_filename)
onnx.checker.check_model(model)
```
To build the TRT Engine, I run 
```
./trtexec --onnx=<onnx_model_path> --verbose
```
The error that I receive when running trtexec is the following (see above repo for full standard error outputs): 
```
[08/02/2021-15:12:06] [E] [TRT] ModelImporter.cpp:726: ERROR: ModelImporter.cpp:179 In function parseGraph:
[6] Invalid Node - Resize_62
[graph.cpp::computeInputExecutionUses::519] Error Code 9: Internal Error (Floor_49: IUnaryLayer cannot be used to compute a shape tensor)
```

**Comment by lgg** [comment]
@Alex-Derhacobian hello, did you solve the issue?

**Comment by liuyiyiyiyi** [comment]
Did you solve the problem?

**Comment by nvpohanh** [comment]
also, this may have been fixed in TRT 8.4.1 (8.4 GA).
